x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/action_delete.test.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { cleanup, queryHelpers, render } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import * as CheckPrivilige from '../../../../../privilege/check_privilege';
+
+import { DeleteAction } from './action_delete';
+
+import mockAnalyticsListItem from './__mocks__/analytics_list_item.json';
+
+jest.mock('../../../../../privilege/check_privilege', () => ({
+  checkPermission: jest.fn(() => false),
+  createPermissionFailureMessage: jest.fn(),
+}));
+
+const queryByTestSubj = (container: HTMLElement, id: string) =>
```

Comment:
might be a reusable test util

Reply:
yes that might be useful moved it to `ml/public/util/test_utils.ts` in 5bab66f.